pypeit/metadata.py

```diff
@@ -223,202 +228,88 @@ def get_meta_data_model():
         # Return
         return meta_data_model
 
-    def _add_bkg_pairs(self, data, bkg_pairs):
-        """
-        Add the background-pair columns to the table.
-        
-        This include two columns called `comb_id` and `bkg_id` that
-        identify object and background frame pairs.  The string
-        indicates how these these columns should be added::
-
-            - `empty`: The columns are added but their values are all
-              originally set to -1.  **This is currently the only
-              option.**
+#    def _add_bkg_pairs(self, data, bkg_pairs):
```

Comment:
Just checking:  `calib_id` and `bkg_id` don't show up for 
multi-slit, right?  Unless requested.

Reply:
Right, I didn't change `set_pypeit_cols`.  These columns are always added to `fitstbl` just like before, but only output if `write_bkg_pairs` is true.